src/libZilliqa/Zilliqa.cpp

```diff
@@ -236,6 +236,7 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     if (!AccountStore::GetInstance().RefreshDB()) {
       LOG_GENERAL(WARNING, "AccountStore::RefreshDB failed");
     }
+    AccountStore::GetInstance().PurgeUnnecessary();
```

Comment:
Is this function meant to be called for `seedpubs or seedprvs or seednodes`?

Reply:
No